.gitlab/.gitlab-ci.yml

```diff
@@ -174,7 +194,7 @@ build featurebase:
 build amd container fb:
   stage: test
   tags:
-    - shell
+    - gdavis-test-shell
```

Comment:
Ia this the name we want to use?

Reply:
That was done just to force running this on a one-off gitlab-runner that I made. https://github.com/molecula/molecular-terraform/pull/307 needs to be merged after this.